nncf/quantization/algorithms/weight_compression/openvino_backend.py

```diff
@@ -131,38 +133,128 @@ def set_weight(
 
         del const_node
 
+    def insert_adapters(self, wc_params: WeightCompressionParameters, lora_A: Tensor, lora_B: Tensor, int8_lora: bool):
+        input_node = self.name_to_node_mapping[wc_params.node_with_weight.node_name].input_value(0)
+        activation_dtype = input_node.get_element_type()
+        should_add_convert_node = activation_dtype != ov.Type.f16
+        mm_node = self.name_to_node_mapping[wc_params.node_with_weight.node_name]
+
+        if int8_lora:
+            a_wc_params = deepcopy(wc_params)
+            a_wc_params.compression_config = WeightCompressionConfig()
```

Comment:
I believe this is a workaround for the bad function signature of `_create_compression_subgraph`.